Source/WebKit/Shared/unix/BreakpadExceptionHandler.cpp

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2020 RDK Management
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#if ENABLE(BREAKPAD)
+#include "config.h"
+#include "BreakpadExceptionHandler.h"
+
+#include <client/linux/handler/exception_handler.h>
```

Comment:
Weird include path. It seems like a clear error in breakpad.pc.in and breakpad-client.pc.in, [here](https://github.com/google/breakpad/blob/defdcb714e0c8113ade899e416d3024ac8024548/breakpad-client.pc.in#L4). `includedir=@includedir@/@PACKAGE_NAME@` should be `includedir=@includedir@`, that way you `#include <breakpad/client/linux/handler/exception_handler.h>` instead of the very confusing `<client/linux/handler/exception_handler.h>`. I wonder if Google would be interested in fixing that, at the cost of breaking current API users.

Anyway, I think you can get away with including it with writing `#include <breakpad/client/linux/handler/exception_handler.h>` anyway, since the top-level include directory will inevitably be in the include path regardless.